components/BarcodeScanner.vue

```diff
@@ -39,6 +39,10 @@ export default {
     showQrBox: {
       type: Boolean,
       default: false
+    },
+    lightTorch: {
```

Comment:
This name seems a bit awkward to me and it doesn't seem to clearly capture that it's just used as the default value for the first time the component is mounted. But that sort of feature itself seems like a pretty limited / odd feature (i.e., like it was made to quickly solve a need, but isn't a good longer term feature). Do we instead intend to watch this value / make it reactive so that it could programmatically turn the torch off and on while the scanner is active?

Even if we don't do that now, it seems like something that might happen later and then we'd have to have two different properties or another breaking change. Can we do this name instead?:

```suggestion
    torchOn: {
```

That way, even if it's not reactive in this release, making it so in the future seems like an easy change without a breaking release.

Reply:
> Do we instead intend to watch this value / make it reactive so that it could programmatically turn the torch off and on while the scanner is active?

No, that wasn't the intent of the PR. I didn't think we wanted to be able to control the torch from outside of the component. This PR was meant to only affect the component on mount.

The previous behavior was to always light the torch when the component was mounted. This was done because  having the torch on helped when we had the older scanner (which had poor performance). Now that we're using native scanners, the torch is rarely helpful and often puts glare on plastic cards and screens such that you have to repeatedly turn the torch off to get a good scan (and keep doing that on every scan). The usability of "torch on by default" has not been good, so we want to keep it off by default (but allow for use cases where that's not the best decision). There might be use cases where you might want to always turn the torch on by default (you always expect to use the app in dimly lit spaces -- or there is an app setting where the user has noted they want to override default initialization behavior).

I was also concerned that other things might affect the torch from outside of the scanner component (like if the person set it on via the phone's native controls)... so, the PR was more about synchronizing to the default state requested by the application developer rather than toggling the state, whatever it might be.

All that said, we can do the rename -- I'm indifferent between the two names, but wanted to make sure we're aligned on the purpose of the property.